client/state/sites/reducer.js

```diff
@@ -103,8 +103,6 @@ export function items( state = {}, action ) {
 			}, initialNextState );
```

Comment:
I've not caught up to the specific requirements, but speaking generally, I think we'd want sites in states to be kept in sync with updates as they're received.